test/datalog_tests/simple.dl

```diff
@@ -1151,6 +1152,16 @@ function any(xs: Vec<bool>): bool =
     return res
 }
 
+// A weird implementation of any using continue inside return.
+function weird_any(xs: Vec<bool>): bool =
+{
+    var res = false;
+    for (x in xs) {
+        return if (x == true) { true } else { continue }
```

Comment:
I don't think I have seen this kind of stuff before in any language.
People may find it confusing, but if it works as expected for the "normal" cases perhaps it's fine.

Reply:
Rust is ok with this. We can of course restrict the use of constructs that interrupt normal control flow (`continue`, `break`, and `return`) to avoid weird cases, but "weird" is hard to define, and oftentimes proves useful in practice...